docs/cookbook/html-conversion.md

```diff
@@ -74,9 +74,9 @@ npm install bootstrap
 This template uses Bootstrap 5, which is the latest version as of this writing. If you need another version for your template, make sure to specify it during the install.
 :::
 
-The next thing we will do is copy the contents of the `scss` folder into our project. There are multiple places that we could add these files, but the best place is in a project-level `modules/asset` folder. If the folder doesn't already exist in your project, create it. Next, within that folder create a `ui/src` folder and copy the entirety of the `dist/scss` folder.
+The next thing we will do is copy the contents of the `scss` folder that contains all of the theme-specific styling into our project. There are multiple places that we could add these files, but the best place is in a project-level `modules/asset` folder. If the folder doesn't already exist in your project, create it. Next, within that folder create a `ui/src` folder and copy the entirety of the `dist/scss` folder.
```

Comment:
I think it would be better to say that these files could be added in any Apostrophe module, but we recommend creating an "asset" module just for your project, e.g. a "modules/asset" folder. To lightly emphasize that Apostrophe is made up of modules, without getting too deep in those weeds.

The user also has to activate that module by adding it to `app.js` in order for it to be recognized and have any effect.

Reply:
Changed the text to clarify and make the extra point.